nixos/roles/ceph/rgw.nix

```diff
@@ -129,30 +129,37 @@ in
       networking.firewall.extraCommands =
         let
           srv = fclib.network.srv;
-        in
-        ''
-          set -x
-          # Accept traffic from S3 gateways from within the SRV network.
-          ip46tables -w -t nat -N fc-nat-pre
-
-        '' + (lib.concatMapStringsSep "\n"
-          (net: ''
-            iptables -A nixos-fw -i ${srv.interface} -s ${net} -p tcp --dport 80 -j ACCEPT
-            # PL-130368 Fix S3 presigned URLs
-            iptables -t nat -A fc-nat-pre -p tcp --dport 7480 -j REDIRECT --to-port 80
+          sto = fclib.network.sto;
+        in lib.mkMerge [
+          (lib.mkOrder 700 ''
+            # Ensure that conntrack is enabled for RGW connections using port redirects
+            iptables -w -t raw -A fc-raw-prerouting -i ${sto.interface} -p tcp --dport 7480 -j RETURN
+            iptables -w -t raw -A fc-raw-output -o ${sto.interface} -p tcp --sport 80 -j RETURN
```

Comment:
this is v4 only, is that on purpose?

Reply:
are we only suppressing connection tracking on v4?

Reply:
> are we only suppressing connection tracking on v4?

Yes, that's correct. When you turned off the connection tracking you only did so for v4.

Reply:
Fixed in 3cfa511